iotedgedev/modules.py

```diff
@@ -27,16 +29,20 @@ def build(self):
 
                 self.output.info("BUILDING MODULE: {0}".format(module_dir))
 
-                # Find first proj file in module dir and use it.
-                project_files = [os.path.join(module_dir, f) for f in os.listdir(
-                    module_dir) if f.endswith("proj")]
+                # if no module.json found ? default to .net
+                json_file_path = os.path.join(os.getcwd(), "modules", module,"module.json")
```

Comment:
module_dir has the path to module, you need to do os.path.join(module_dir, "module.json")